cmd/infra/powervs/create.go

```diff
@@ -168,6 +183,7 @@ func NewCreateCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&opts.BaseDomain, "base-domain", opts.BaseDomain, "IBM Cloud CIS Domain")
 	cmd.Flags().StringVar(&opts.ResourceGroup, "resource-group", opts.ResourceGroup, "IBM Cloud Resource Group")
+	cmd.Flags().StringVar(&opts.Namespace, "namespace", opts.Namespace, "A namespace to contain the generated resources")
```

Comment:
why does infra creation requires a namespace?

Reply:
We are generating the secrets which are going to get installed in management namespace during cluster deployment in Infra creation itself and passing those secrets here https://github.com/openshift/hypershift/blob/ba156a10e8441dfd6d3fbeaeed94b8cbcf06bf15/cmd/cluster/powervs/create.go#L169 and here https://github.com/openshift/hypershift/blob/ba156a10e8441dfd6d3fbeaeed94b8cbcf06bf15/api/fixtures/example.go#L339 so those can be directly deployed in management namespace.

Only expectation is user need to pass same namespace (or not passing) option if they are creating the infra and cluster separately.

Since few details require for this secret generation is not available in create cluster & destroy cluster scope, so moved these generating secrets to deleting secrets operation to create infra and destroy infra scope.